peer/test_utils.go

```diff
@@ -38,6 +38,10 @@ const (
 	// timeout is a timeout value to use for tests which need to wait for
 	// a return value on a channel.
 	timeout = time.Second * 5
+
+	// cltvRejectDelta is the minimum delta between expiry and current
+	// height below which htlcs are rejected.
+	cltvRejectDelta = 13
```

Comment:
nit: since this is compiled into the `htlcswitch` package, I'd recommend perfixing it with `test` to avoid faulty reuse later.

Reply:
Good point, fixed.